### PR TITLE
[ODH-2190]: Airflow Manager & Worker DAG executions

### PR DESCRIPTION
**Pull request by nilan3**
Basic analytics job to extract airflow manager scheduler metrics in tadem with airflow work dag execution metrics in order to build dashboards similar to http://odh.obo.lgi.io/grafana/dashboard/db/be-airflow-dashboard-v1?orgId=1

### CODE REVIEW

File: configuration/basic_analytics/airflow/worker_manager_dag_exec_configuration.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+spark:
```
Comment (needmorec) [inline configuration/basic_analytics/airflow/worker_manager_dag_exec_configuration.yml]: we removed this folder, you can keep it locally and add to .gitignore

File: src/applications/basic_analytics/airflow/manager_worker_dag_exec.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        :param data_stream: input stream
+        """
+        tenant = self._component_name.split(".")[2]
+        regex_filter_udf = udf(self.regex_filter, BooleanType())
+        regex_extract_udf = udf(self.regex_extract, ArrayType(StringType()))
+        remove_prefix_udf = udf(self.remove_tenant_prefix, StringType())
+
+        airflow_manager = stream \
+            .where(col("topic").isNotNull()) \
+            .filter(regex_filter_udf(col("message"))) \
```
Comment (alimaken) [inline src/applications/basic_analytics/airflow/manager_worker_dag_exec.py]: filter == where. Use one or the other for code consistency.

File: src/applications/basic_analytics/airflow/manager_worker_dag_exec.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        :param x: msg string
+        :return: group matched list
+        """
+        regexs = [r"Created.*DagRun\s(?P<tenant>\w{2,3})_(?P<dag>.*)\s@",
+                  r"Marking.*DagRun\s(?P<dag>.*)\s@.*(?P<tenant>\w{2,3})-crid.*\s(?P<type>\w+)$",
+                  r"Marking.*DagRun\s(?P<tenant>\w{2,3})_(?P<dag>.*)\s@.*\s(?P<type>\w+)$"]
+
+        for r in regexs:
+            match = re.search(r, x)
+            if match:
```
Comment (alimaken) [inline src/applications/basic_analytics/airflow/manager_worker_dag_exec.py]: What happens if there is no match? What will happen to the subsequent call `col("message").getItem(0)`

File: configuration/basic_analytics/airflow/worker_manager_dag_exec_configuration.yml (hunk 1 above)
Comment (alimaken) [reply configuration/basic_analytics/airflow/worker_manager_dag_exec_configuration.yml]: @needmorec I don't understand what you are talking about.

File: src/applications/basic_analytics/airflow/manager_worker_dag_exec.py (hunk 3 above)
Comment (nilan3) [reply src/applications/basic_analytics/airflow/manager_worker_dag_exec.py]: @alimaken there should be a match as there is a filter before this step.

Comment (alimaken) [reply src/applications/basic_analytics/airflow/manager_worker_dag_exec.py]: OK.

**Pull request merged**